### ISSUE TITLE
Darker background on web-based project player

### ISSUE THREAD

**Issue by vazquez** [mozilla/webmaker-core#418]
From @flukeout's ideation:

> Besides the darker background and more prominent navigation arrows, I also added a space between the top and bottom of the page and the edge of the browser. I think this makes it feel more like it's part of a grid. When the page is 100% height it feels mostly like a side-to-side viewer. 

![_mvfhkcbaoeg6panqm_le-jxlsk3mko1vtnihla-tc0](https://cloud.githubusercontent.com/assets/1119821/8913056/9d95aeda-3464-11e5-82eb-1fc4591062f1.png)

**Comment by vazquez** [comment]
@flukeout -

Check this out:

#### Desktop Player

![desktop - page view dark](https://cloud.githubusercontent.com/assets/1119821/8916234/2eaa0f48-3478-11e5-94bc-ef109533f373.png)

#### Desktop Player Short View

![desktop - page view short dark](https://cloud.githubusercontent.com/assets/1119821/8916237/326d9578-3478-11e5-8ad0-f60a304dbd5b.png)

#### Mobile Web Player

![mobile - page view dark](https://cloud.githubusercontent.com/assets/1119821/8916250/45d4f052-3478-11e5-997a-df60a7a90309.png)

#### Mobile Web Player Short View

![mobile - page view short dark](https://cloud.githubusercontent.com/assets/1119821/8916246/407ecf7e-3478-11e5-86f0-1e0da728e42c.png)

**Comment by vazquez** [comment]
@flukeout - 

Improvement on Mobile Web Player

![mobile - page view short dark 2](https://cloud.githubusercontent.com/assets/1119821/8916998/5be7939a-347d-11e5-8940-b08346ff3f20.png)

I then thought it looked almost like a ticket stub, so I moved out the top and bottom arrows:

![mobile - page view short dark copy](https://cloud.githubusercontent.com/assets/1119821/8917007/6fe7e804-347d-11e5-8476-ecc28e2ad61e.png)

**Comment by flukeout** [comment]
@vazquez Great work on these! :cake:

I like both versions of the desktop player the last version of the Mobile player. We'd need to make sure that the side arrows are easy to tap (they have a larger, invisible tap area) because they are quite slim.

Eventually we'll have swiping back so the tap targets won't be as important.

**Comment by flukeout** [comment]
Oh - can you send me a link to the source files?

**Comment by vazquez** [comment]
I also like the last version of the Mobile Player :smile: 

The file is under Google Drive. **`webmaker-android > Version 2.0 > Webmaker-Player-UI-2015-07-27.sketch`**

The dark iterations are on the far right.

**Comment by vazquez** [comment]
Addressing feedback from yesterday's critique.

Changes made:
- Reduced chunkiness of arrows.
- Added depth treatment to arrows
- CC Icon displays on left bottom
- CC icon has better visual treatment
- Aligned header

![desktop - page view dark](https://cloud.githubusercontent.com/assets/1119821/8989175/83f57874-36b8-11e5-8b83-41866fd36a38.png)  

![mobile - page view short dark copy](https://cloud.githubusercontent.com/assets/1119821/8989174/83f547b4-36b8-11e5-87db-f0b0c162c654.png)

**Comment by vazquez** [comment]
@flukeout - Let me know what you think :arrow_up:

**Comment by flukeout** [comment]
Added the two screens above to redpen for commenting...
https://redpen.io/p/qy2e9e018a55460189

**Comment by vazquez** [comment]
@flukeout - Can you give me upload powers on RedPen?

In the meantime, check out these edits :sunglasses: 

![desktop - page view dark](https://cloud.githubusercontent.com/assets/1119821/9013546/a129ce2c-378a-11e5-9223-46de571b912b.png)  

![mobile - page view dark](https://cloud.githubusercontent.com/assets/1119821/9013547/a134b972-378a-11e5-8327-61c4305b5452.png)

**Comment by flukeout** [comment]
@vazquez Powers granted!

**Comment by vazquez** [comment]
Thanks man :) I updated the Red Pen

**Comment by vazquez** [comment]
@flukeout - Redpen has been updated with the feedback. Waiting for your R+

**Comment by xmatthewx** [comment]
:+1:  - looks way better. 

@vazquez - can you summarize changes between this and current build?

**Comment by vazquez** [comment]
#### UI Changes [browser-based player]
1. In Player view, the background is now `#444444`
2.  Neighbours are now `10%` opacity instead of `7%`
3. Author (i.e. "by lukepacholski") has a `50%` opacity. Project title still displays at `100%` opacity.
4. The nubs have a different visual treatment. They are smaller and `#FFF` with a `70%` opacity. They also display a box-shadow.

![desktop - page view dark](https://cloud.githubusercontent.com/assets/1119821/9087343/0cabb3ce-3b58-11e5-94d8-1e2bb8cc4590.png)

---

The same changes apply to mobile view as well. However, there is one significant change in this new iteration of mobile view:
1. The page size has diminished to make room for nubs (specifically the left and right nubs). We have sacrificed page real estate but we feel this frames the page very nicely.

![mobile - page view dark](https://cloud.githubusercontent.com/assets/1119821/9087771/afbbb95e-3b5a-11e5-81c5-d0e03e3cc900.png)

**Comment by xmatthewx** [comment]
This looks sharp. Thanks for the notes. Half-moons are out altogether then, yes?

**Comment by vazquez** [comment]
@xmatthewx - 

For mobile view, half moons are out completely.

For desktop browser, I have been toying with the idea that if the browser's height become short, then we indent the top and bottom neighbours.

![desktop - page view short dark](https://cloud.githubusercontent.com/assets/1119821/9088757/b70bb326-3b5f-11e5-9573-a07e7b98f3ba.png)

What do you think about something like this?

**Comment by xmatthewx** [comment]
It looks fine, but for simplicity we should keep it consistent everywhere. 

In the current implementation, the left/right arrows are at the browser's edge rather than next to the center page. Let's make sure that gets corrected during this theme update.

**Comment by vazquez** [comment]
+1 I agree about keeping consistency everywhere.

I will make sure to note the placement of left and right arrows.

Can we call this R+? I will file a subsequent implementation ticket.

**Comment by xmatthewx** [comment]
:+1:
